ts/Maps/MapNavigation.ts

```diff
@@ -240,6 +241,25 @@ MapNavigation.prototype.update = function (
                 height: 2 * button.height
             });
 
+            if (button.element) {
+                const expOptions = chart.options.exporting,
+                    expBtn = (expOptions && expOptions.buttons && expOptions.buttons.contextButton);
+
+                // If the nav buttons are on the same side
+                if (expBtn && defined(buttonOptions.y) && chart.exportingGroup) {
+                    if (chart.options && chart.title &&
+                        chart.title.textStr === '' &&
```

Comment:
What about one title or subtitle is enabled?

Reply:
If either has text, the bug is gone, so we check if both are empty

Reply:
An edge case when this is not true - http://jsfiddle.net/BlackLabel/rLcyx5jw/1/

Reply:
@DJTechnoo ☝️